src/components/Profile View/ProfileForm.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react"
+import AxiosAuth from "../Auth/axiosWithAuth"
+
+const ProfileForm = ({ store }) => {
+  const [input, setInput] = useState({
+    businessName: "",
+    ownerName: "",
+    address: "",
+    secondAddress: "",
+    city: "",
+    state: "",
+    zipcode: "",
+    hours: "",
+    curbHours: "",
+  })
+
+  const handleChange = (e) => {
+    setInput({
+      [e.target.name]: e.target.value,
+    })
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    AxiosAuth()
+      .put(`https://shopping-cart-be.herokuapp.com/api/store/`, input)
+      .then((res) => console.log("ProfileFormSubmit", res.data))
```

Comment:
Is there a reason for there only being a console.log in your .then?

Reply:
No reason except for seeing the if data is changed. Since it is a put call there is not much write in .then call except for console logging.